elastic/elastic-agent
# [Update Agent Version] replace 8.x by master on `mage integration:updatePackageVersion`

## What does this PR do?

replace 8.x by main on `main.Integration.UpdatePackageVersion`

## Why is it important?

It's necessary to fetch the latest snapshots for the 8.x branch

## Checklist




## Disruptive User Impact

None

## How to test this PR locally

merge this on your local 8.x branch
run:
 - `mage integration:updatePackageVersion`
 - `mage integration:updateVersions`

you should see:
```sh
❯ mage integration:updatePackageVersion
8.16.0
 ~/devel/github.com/elastic/elastic-agent   8.x ⇡1  19:09:29                                                                                                                    8.342s
❯ mage integration:updateVersions
{
  "UpgradeToVersion": "8.16.0",
  "CurrentMajors": 1,
  "PreviousMajors": 1,
  "PreviousMinors": 2,
  "SnapshotBranches": [
    "8.x",
    "8.15",
    "7.17"
  ]
}
```

and a git diff like
```
diff --git a/.agent-versions.json b/.agent-versions.json
index 9acfaf293b..cc13621efd 100644
--- a/.agent-versions.json
+++ b/.agent-versions.json
@@ -1,9 +1,9 @@
 {
   "testVersions": [
-    "8.15.0",
-    "8.15.0-SNAPSHOT",
-    "8.14.4-SNAPSHOT",
+    "8.15.2-SNAPSHOT",
+    "8.15.1",
     "8.14.3",
-    "7.17.24-SNAPSHOT"
+    "7.17.25-SNAPSHOT",
+    "7.17.24"
   ]
 }
```

## Related issues


- N/A

## Questions to ask yourself

- How are we going to support this in production?
- How are we going to measure its adoption?
- How are we going to debug this?
- What are the metrics I should take care of?
- ...
